src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -5,22 +5,27 @@
 import static org.folio.circulation.domain.ItemStatus.CHECKED_OUT;
 import static org.folio.circulation.domain.ItemStatus.IN_TRANSIT;
 import static org.folio.circulation.domain.ItemStatus.MISSING;
+import static org.folio.circulation.domain.representations.ItemProperties.CONTRIBUTORS;
```

Comment:
Contributors are part of an instance not an item, why is this using the item properties definition?

Reply:
I missed that. Is it ok to create `InstanceProperties` for this key?

Reply:
Yes, that is ok, please do

Reply:
Done